Neo4j.Driver/Neo4j.Driver/Internal/Protocol/BoltProtocolVersion.cs

```diff
@@ -99,6 +105,11 @@ public bool Equals(BoltProtocolVersion rhs)
         return _compValue == rhs._compValue;
     }
 
+    public int CompareTo(BoltProtocolVersion other)
+    {
+        return other == null ? 1 : _compValue.CompareTo(other._compValue);
```

Comment:
A comment would be nice on this line since the 1 is a magic number

Reply:
This method isn't actually used, looks like I left it in by accident. Removing.